### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by AbuRayhaan**
##Viable-Project: Project Setup

In this milestone, I did the following:
- Added linters file for `html-css-js` files.
- Added the `index.html` file.
- Added the `style.css` file.
- Added the `script.js` file.
- Loom video link: https://www.loom.com/share/627f866e8c9f4661903c8d8c10785f75

### CODE REVIEW

**Review by Kingstalux — COMMENTED** [review]
Hi @AbuRayhaan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f644550476b6e5269456e6965632f67697068792e676966


## Invalid Code Review Request 🛑 

- [ ] Kindly add a link to the video presentation of your capstone project which should have a maximum length of 5 minutes where you present all the features of your project and talk about a piece of interesting code you worked on 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                    <li class="header-menu-item"><a href="#work">English</a></li>
+                    <li class="header-menu-item"><a href="#about">My Page</a></li>
+                    <li class="header-menu-item"><a href="#contact">Login</a></li>
+                </ul>
+            </div> 
+        </nav> 
+        <nav class="navbar-white">
+            <div>
+                <img src="images/vg-logo.png" alt="VG Logo">
+            </div>
```
Comment (ShoiraTa) [inline index.html]: - [ ] Please make sure that the logo is a link. Users should be able to click on logo and get to homepage.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            <ul class="header-white-menu">
+                <li class="header-white-menu-item"><a href="#about">About</a></li>
```
Comment (ShoiraTa) [inline index.html]: - [ ] When I click on `About` link, a new page should open with a `footer` and its own `navbar`, but instead I am taken to the different section of the same page with `#about` id. I noticed that in project you have only 1 html index file. Please kindly add `about.html` file as well and make sure that the About link takes to `about.html` page. 
- [ ] While writing `about.html` please try to stick  to provided [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                <!--Dynamically Generated-->
+            </div>
+        </section>
+
+        <section class="about" id="about">
+            <h1 class="headline-hello">Achievers world</h1>
+            <h2 class="about-summit"> VIBRANT AFRICANS' YOUTH SUMMIT 2022</h2>
+            <p class="about-para">A global network of professional Software Developers 
+                aimed at sharing opportunities and creating a positive change 
+                with people from over 80 countries is taking place in Nigeria.
```
Comment (ShoiraTa) [inline index.html:81]: - [ ]  Based on project requirements it is necessary to use font families `Cocogoose, Lato`  as provided in the theme:
Screenshot 2022-04-13 at 17 45 41

File: index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+            <h2 class="headline-summit"> VIBRANT AFRICANS'<br>YOUTH SUMMIT 2022</h2>
+            <p class="headline-para">A global network of professional Software Developers 
+                aimed at sharing opportunities and creating a positive change 
+                with people from over 80 countries is taking place in Nigeria.
+            </p>
+            <p class="headline-venue">@Eko Hotel and Suites, Victoria Island, Lagos, Nigeria.</p>
+        </section>
+
+        <section class="program" id="program">
+            <h1 class="main-program">Main Program</h1>
```
Comment (ShoiraTa) [inline index.html:58]: - [ ] Kindly set the same height for the cards of this section: 
Screenshot 2022-04-13 at 20 17 32

File: index.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+            <hr class="line">
+            <div class="partner-list">
+                <img src="images/logo-jekaplay.png" alt="jekaplay">
+                <img src="images/logo-afrotainment.PNG" alt="afrotainment">
+            </div>
+        </section>
+
+        <footer class="footer-section" id="sponsor">
+            <h5 class="copyright">2022 Viable Group Conference Copy Rights Reserved.</h5>
+        </footer>
```
Comment (ShoiraTa) [inline index.html]: - [ ] Please make sure that footer contains a logo aligned to the left and text aligned to the right. Currently I can only see centred text: 
Screenshot 2022-04-13 at 20 21 30

File: README.md [hunk 6]
```diff
@@ -1 +1,57 @@
...
+To get a local copy up and running:
+
+1. Clone this repository or download the Zip folder:
+
+**``git clone https://github.com/AbuRayhaan/Mobile-Website``**
+
+1. Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd <folder>``**
+
```
Comment (ShoiraTa) [inline README.md]: - [ ] Please update this section with current project links

File: README.md [hunk 7]
```diff
@@ -1 +1,57 @@
...
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐ if you like this project and how we manage to build it!
+
+## 📝 License
+
+This project is [MIT](./MIT.md)
```
Comment (ShoiraTa) [inline README.md]: - [ ] License link is broken, kindly add `MIT` License file  to root directory to make the link work

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @AbuRayhaan ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — COMMENTED** [review]
Hi @AbuRayhaan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**